README.md

```diff
@@ -82,8 +92,49 @@ Create an `index.html` file into `target/generated` directory and add the follow
 </html>
 ```
 
-Now run a web server locally inside the `target/generated` directory to see the `hello-triangle` in the browser.
-e.g. `python -m http.server`
+Now run a web server locally inside the `target/generated` directory to see the `hello-triangle` in the browser. Examples of servers are rust's 
```

Comment:
We can share the web server instructions between WebGPU and WebGL

Reply:
The reason I didn't include it, and once we split between the two, I fully expected someone to stop reading once the WebGPU section and then complain that they can't figure it out. Once we move it to the wiki where there is a table of contents to signal people to keep reading, we can deduplicate it.